pkg/tfgen/docs.go

```diff
@@ -965,6 +970,10 @@ func parseAttributesReferenceSection(subsection []string, ret *entityDocs) {
 	}
 }
 
+func flattenListAttributeKey(attribute string) string {
+	return strings.ReplaceAll(attribute, ".0", "")
+}
```

Comment:
Where do these lists come from? Are they always `.0`, never `.N` or `.1`?

Reply:
This is an example of a [TF schema.TypeList](https://github.com/hashicorp/terraform-provider-aws/blob/main/internal/service/mq/broker.go#L175) which I believe only ever uses the `.0` to access the list elements. I'm admittedly a bit fuzzy on this so I'm happy to be corrected here.
I also searched the p-aws schema as well as upstream/website for any indications of !=`.0` and came up with nothing. I know that's not proof and if you want to make this digit-agnostic, we can.

Reply:
I trust your check. This is the type of information that would make an excellent comment on line 974.